composer.json

```diff
@@ -14,7 +14,7 @@
         "php": ">=5.3",
         "react/event-loop": "0.3.*",
         "react/promise": "~1.0",
-        "clue/socket-react": "0.2.*",
+        "clue/socket-react": ">=0.2",
```

Comment:
Using unbounded version constraints is really dangerous and will probably break in the future. Better use something like `^0.2|^0.3` if you know that `0.3.0` didn't break your code.